Specs/DateTime/English/MergedParser.json

```diff
@@ -793,45 +794,26 @@
     ]
   },
   {
-    "Input": "Set an appointment for Easter Sunday",
+    "Input": "Set an appointment for Easter Monday 2017",
```

Comment:
It seems this case behaves different between the VSO build (it passes) and the AppVeyor one (it fails). We'll debug why.

Reply:
Please take a look. With latest commit, my local build passed but PR build failed once more. Thx